src/store/dao/actions.js

```diff
@@ -1,10 +1,11 @@
 /* eslint-disable no-unreachable */
 import camelToSnakeCase from '~/utils/camelToSnakeCase'
+import HyphaTokensSaleUtil from '@hypha-dao/hypha-token-sales-util'
 
-export const createDAO = async function (context, { data }) {
+export const createDAO = async function (context, { data, isDraft }) {
   const actions = [{
     account: this.$config.contracts.dao,
-    name: 'createdao',
+    name: isDraft ? 'createdaodft' : 'createdao',
```

Comment:
@arsenijesavic contract action names should be stored as constants and reused, instead of hardcoded strings.